design_system/lib/src/widgets/cat_widget.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+import '../../design_system.dart';
+
+class GrumpCatImage extends StatelessWidget {
+  @override
+    Widget build(BuildContext context) {
```

Comment:
Using this widget inside the main_app will not break the test because the it is used internally